### PR TITLE
Gives Resist button in HUD to monkeys and diona nymphs

### LINKED ISSUE #4734: Diona nymphs can resist, but lack a resist button in the HUD
You can resist by using the verb, you just lack a convenient button.

### PR DESCRIPTION
**Pull request by 9600bauds**
Fixes #4734
This only adds a HUD button, monkeys and diona nymphs could already resist before by using the verb.

### CODE REVIEW

File: code/_onclick/hud/monkey.dm [hunk 1]
```diff
@@ -127,6 +127,14 @@
 	using.layer = 19
 	src.adding += using
 
+	using = getFromPool(/obj/screen)
+	using.name = "resist"
+	using.icon = ui_style
+	using.icon_state = "act_resist"
+	using.screen_loc = ui_pull_resist
+	using.layer = 19
```
Comment (ComicIronic) [inline code/_onclick/hud/monkey.dm]: But why?

**Comment by ComicIronic** [comment]
What does this look like in the HUD?

File: code/_onclick/hud/monkey.dm (hunk 1 above)
Comment (9600bauds) [inline code/_onclick/hud/monkey.dm]: It's exactly like that on human.dm and everything looks exactly the same so I'm not sure why what

**Comment by 9600bauds** [comment]
As in the human HUD, since the monkey HUD still has the same drop and pull etc buttons

**Pull request merged**